code/datums/uplink_item.dm

```diff
@@ -220,6 +220,16 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 5
 	job = list("Chief Medical Officer", "Medical Doctor", "Geneticist", "Psychiatrist",	"Chemist", "Paramedic", "Coroner", "Virologist")
 
+//Virology
+
+/datum/uplink_item/jobspecific/rad_laser
+	name = "Viral Injector"
+	desc = "A modified hypospray disguised as a functional pipette. The pipette can infect victims with viruses upon injection."
+	reference = "RL"
+	item = /obj/item/reagent_containers/dropper/precision/viral_injector
+	cost = 3
+	job = list("Virologist")
```

Comment:
`hijack_only = TRUE`
Add this on the end, as you state it is only going to be used for lethal viruses in hijack situations.

Reply:
Actually I think the point is to enable lethal virus assassinations with deadly virus that don't spread for non-hijack traitors

Reply:
Yeah, I just realised. Whoops.